src/activate/activate.c

```diff
@@ -81,8 +81,8 @@ static int open_sockets(int *epoll_fd, bool accept) {
          *  would be a pain to fix.
          */
 
-        STRV_FOREACH(address, arg_listen) {
-                fd = make_socket_fd(LOG_DEBUG, *address, arg_socket_type, (arg_accept*SOCK_CLOEXEC));
+        STRV_FOREACH (address, arg_listen) {
```

Comment:
Hmm, i would have said a space is more reasonable, since the STRV_FOREACH behaves more like a loop than like a macro (is indented and not on the left, affects other code by indenting).

Reply:
i'd really prefer if there was no space added here, i agree with @yuwata... these macros are indeed a bit like loops, but they are loops that also call functions, so they are something in the middle... But it's indeed a difficult case, but I think it would be better to avoid the spaces

Reply:
OK, i think this cannot be done with clang-format right now, so this will be a canditate for the tools/post-clang-format.php